halo2_proofs/src/poly/commitment/prover.rs

```diff
@@ -33,23 +33,23 @@ pub fn create_proof<
     params: &Params<C>,
     mut rng: R,
     transcript: &mut T,
-    px: &Polynomial<C::Scalar, Coeff>,
-    blind: Blind<C::Scalar>,
+    p_poly: &Polynomial<C::Scalar, Coeff>,
+    p_blind: Blind<C::Scalar>,
     x: C::Scalar,
 ) -> io::Result<()> {
```

Comment:
In general, make it clearer how this code corresponds to the protocol description. It seems to start off by sampling s which is at step 20 of the protocol description; it's not obvious where the rest of the protocol before that is implemented.